io.c

```diff
@@ -8557,6 +8557,12 @@ rb_io_printf(int argc, const VALUE *argv, VALUE out)
     return Qnil;
 }
 
+VALUE
```

Comment:
We should add a `static` here to make sure it's not publicly visible.

```suggestion
static VALUE
```